autogen

```diff
@@ -161,6 +161,11 @@ Options:
                            add a top-level documentation comment after the
                            license header (default: enabled)
 
+  --no-pycode, --no-pyc
```

Comment:
I think you make a good point overall, but adding a separate flag per-language seems overkill, because it's not possible to generalize it in a config or editor file, for example.

How about a more generic `--no-code` flag that disables all codegen for all languages instead of a Python-specific one?

Reply:
Sound great to me, agreed